app/addons/activetasks/tests/activetasks.componentsSpec.react.jsx

```diff
@@ -0,0 +1,116 @@
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+define([
+  'api',
+  'addons/activetasks/views',
+  'addons/activetasks/resources',
+  'addons/activetasks/components.react',
+  'addons/activetasks/stores',
+  'addons/activetasks/tests/fakeActiveTaskResponse',
+  'react',
+  'addons/activetasks/actions',
+  'testUtils'
+], function (FauxtonAPI, Views, ActiveTasks, Components, Stores, fakedResponse, React, Actions, testUtils) {
+  var assert = testUtils.assert;
+  var TestUtils = React.addons.TestUtils;
+  var activeTasksStore = Stores.activeTasksStore;
+  var activeTasksCollection = new ActiveTasks.AllTasks({});
+  activeTasksCollection.parse(fakedResponse);
+
+  describe('Active Tasks -- Components', function () {
+
+    describe('Active Tasks Polling (Components)', function () {
+      var pollingWidgetDiv, pollingWidget;
+
+      beforeEach(function () {
+        pollingWidgetDiv = document.createElement('div');
+        pollingWidget = TestUtils.renderIntoDocument(React.createElement(Components.ActiveTasksPollingWidget, null), pollingWidgetDiv);
+      });
+
+      afterEach(function () {
+        React.unmountComponentAtNode(pollingWidgetDiv);
+      });
+
+      it('should trigger update polling interval', function () {
+        var spy = sinon.spy(Actions, 'changePollingInterval');
+        var rangeNode = TestUtils.findRenderedDOMComponentWithTag(pollingWidget, 'input');
+        var time = '9';
+
+        TestUtils.Simulate.change(rangeNode, {target: {value: time}});
+        assert.ok(spy.calledOnce);
+      });
+    });
+
+    describe('Active Tasks Table (Components)', function () {
+      var table, tableDiv, spy, filterTab;
+
+      beforeEach(function () {
+        tableDiv = document.createElement('div');
+        activeTasksStore.init(activeTasksCollection.table, activeTasksCollection);
+        table = TestUtils.renderIntoDocument(React.createElement(Components.ActiveTasksController, null), tableDiv);
+
+        // open filter tray
+        filterTab = TestUtils.findRenderedDOMComponentWithClass(table, 'toggle-filter-tab');
+        TestUtils.Simulate.click(filterTab);
+      });
+
+      afterEach(function () {
+        spy.restore();
+        React.unmountComponentAtNode(tableDiv);
+        window.confirm.restore && window.confirm.restore();
+      });
+
+      describe('Active Tasks Filter tray', function () {
+        var radioIDs = [
+          'Replication',
+          'Database-Compaction',
+          'Indexer',
+          'View-Compaction'
+        ];
+
+        it('should trigger change to radio buttons', function () {
+          _.each(radioIDs, function (radioID) {
+            spy = sinon.spy(Actions, 'switchTab');
+            TestUtils.Simulate.change($(table.getDOMNode()).find('#' + radioID)[0]);
+            assert.ok(spy.calledOnce);
+            spy.restore();
```

Comment:
@michellephung what @robertkowalski means here is that you call `spy.restore` in the test and the `afterEach` function. You only need to call it in the `afterEach` function.